sys/build.sh

```diff
@@ -59,24 +59,18 @@ A=$(dirname "$PWD/$0")
 cd "$A" && cd .. || exit 1
 
 if [ "`uname`" = Darwin ]; then
-	DEFAULT_PREFIX=/usr/local
 	# purge previous installations on other common paths
 	if [ -f /usr/bin/r2 ]; then
 		type sudo || NOSUDO=1
 		[ "$(id -u)" = 0 ] || SUDO=sudo
 		[ -n "${NOSUDO}" ] && SUDO=
 		# purge first
 		echo "Purging r2 installation..."
-		./configure --prefix=/usr > /dev/null
+		./configure > /dev/null
```

Comment:
the reason for that is to purge the installation found in /usr before installing further, the problem is that if u have r2 installed from the distro it will basically purge it, so this change makes sense to avoid killing the linux installation.. problem iis when you dont use linux, this will basically remove the previous installation of r2, or basically remove the one from the pkgsystem, because bsd and mac install in /usr/local which iis the default, therefor this is moving the problem from linux to bsd

Reply:
> problem iis when you dont use linux, this will basically remove the previous installation of r2, or basically remove the one from the pkgsystem, because bsd and mac install in /usr/local which iis the default, therefor this is moving the problem from linux to bsd

Linux installs into /usr/local too. This doesn't change anything. `make install` and `make uninstall` will just work. That's the point. FreeBSD installs ports packages to `/usr/local` too (afaik). But the point is without touching `--prefix` it you don't move the problem, you get rid of it.

> because bsd and mac install in /usr/local

Even if so, why is this a problem.